clime/core.py

```diff
@@ -632,7 +632,6 @@ def append_usage(cmd_name, without_name=False):
             usages.append(Command(cmd_func, cmd_name).build_usage(without_name))
 
         usages = []
-        cmd_func = None
```

Comment:
PyFlakes thinks so.

    (clime)tp-mba:clime uranusjr$ pyflakes clime/core.py
    clime/core.py:635: local variable 'cmd_func' is assigned to but never used